src/main/resources/edu/cornell/kfs/module/ld/document/datadictionary/CuYearEndSalaryExpenseTransferDocument.xml

```diff
@@ -18,20 +18,48 @@
    - along with this program.  If not, see <http://www.gnu.org/licenses/>.
  -->
 
-  <bean id="YearEndSalaryExpenseTransferDocument" parent="YearEndSalaryExpenseTransferDocument-parentBean">
+  <bean id="CuYearEndSalaryExpenseTransferDocument" parent="CuYearEndSalaryExpenseTransferDocument-parentBean"/>
+  <bean id="CuYearEndSalaryExpenseTransferDocument-parentBean" abstract="true" parent="TransactionalDocumentEntry">
```

Comment:
Should the parent of CuYearEndSalaryExpenseTransferDocument-parentBean be YearEndSalaryExpenseTransferDocument-parentBean?  It looks like it was prior to this change, so we can keep any  inheritance